nltk/translate/bleu_score.py

```diff
@@ -106,16 +106,17 @@ def sentence_bleu(
     )
 
 
-def corpus_bleu(
-    list_of_references,
-    hypotheses,
-    weights=(0.25, 0.25, 0.25, 0.25),
-    smoothing_function=None,
-    auto_reweigh=False,
+def corpus_bleu_multiple_weights(
+        list_of_references,
+        hypotheses,
+        weights_list,
+        smoothing_function=None,
+        auto_reweigh=False
 ):
     """
     Calculate a single corpus-level BLEU score (aka. system-level BLEU) for all
-    the hypotheses and their respective references.
+    the hypotheses and their respective references for each weighting tuple in
+    weights_list
```

Comment:
This is rather non-normal use of BLEU for >4-grams BLEU, users can already add to the weights list in the original implementation and change the n order.

Reply:
Ah the normal corpus_bleu has been moved down to another function.